controllers/rcj_soccer_team_blue/rcj_soccer_robot.py

```diff
@@ -76,13 +80,59 @@ def get_new_data(self) -> dict:
         return self.parse_supervisor_msg(packet)
 
     def is_new_data(self) -> bool:
-        """Check if there are new data to be received
+        """Check if there is new data from supervisor to be received
 
         Returns:
             bool: Whether there is new data received from supervisor.
         """
         return self.receiver.getQueueLength() > 0
 
+    def parse_team_msg(self, packet: str) -> dict:
+        """Parse message received from team robot
+
+        Returns:
+            dict: Parsed message stored in dictionary.
+        """
+        struct_fmt = 'i'
+        unpacked = struct.unpack(struct_fmt, packet)
+
+        data = {
+            'robot_id': unpacked[0],
```

Comment:
@Adman I guess we could showcase sending the position of a specific robot in here?

Reply:
@mrshu That is showcased in supervisor. And actually sending position doesn't make sense because that information is already sent via supervisor. I believe the documentation demonstrates sending various datatypes clearly. This was meant just as demonstration that it works.

Reply:
Right, I really don't disagree -- I was just thinking if/how could we make it so that this is a bit more useful straight away :)